src/doc/src/reference/unstable.md

```diff
@@ -1665,6 +1666,66 @@ version = "0.1.0"
 workspace = true
 ```
 
+## gc
+
+* Tracking issue: TODO
+
+The `-Zgc` flag enables garbage-collection within cargo's global cache within the cargo home directory.
+This includes downloaded dependencies such as compressed `.crate` files, extracted `src` directories, registry index caches, and git dependencies.
+When `-Zgc` is present, cargo will track the last time any file was used, and then uses those timestamps to manually or automatically delete files that have not been used for a while.
+
+```sh
+cargo build -Zgc
+```
+
+Automatic deletion happens on commands that are already doing a significant amount of work,
```

Comment:
>on

before or after?

Reply:
It runs just after resolution and packages are downloaded, but before compilation. I added a comment to that effect.